### DISCUSSION TITLE
无法查询到Load Profile

### DISCUSSION THREAD

**Discussion by xuwei0912** [apache/doris#25186] [A - General / Q&A] [answered]
已经开启 enable_profile，在Web界面可以看到query的profile，但是没有load任务的，show load profile "/queryid"显示没有queryid任务
version：2.0.2

**Comment by yagagagaga** [comment]
For query, you can execute `show query profile "/queryid"`,
For load, you can execute `show load profile "/queryid"`.

**Comment by xuwei0912** [comment, marked as answer]
fe bug
